ledger/participant-integration-api/src/main/scala/platform/apiserver/AuthServiceConfig.scala

```diff
@@ -26,7 +27,7 @@ object AuthServiceConfig {
   /** [UNSAFE] Enables JWT-based authorization with shared secret HMAC256 signing: USE THIS EXCLUSIVELY FOR TESTING */
   final case class UnsafeJwtHmac256(
       secret: String,
-      jwtTimestampLeeway: Option[JwtTimestampLeeway] = None,
+      override val jwtTimestampLeeway: Option[JwtTimestampLeeway] = None,
```

Comment:
```suggestion
      jwtTimestampLeeway: Option[JwtTimestampLeeway] = None,
```
Case classes don't need override val

Reply:
Same applies throughout this file.